xknxproject/xml/parser.py

```diff
@@ -162,6 +165,19 @@ def parse(self, language: str | None = None) -> KNXProject:
             locations=space_dict,
         )
 
+    def convert_functions(self, functions: XMLFunctions) -> list[str, Functions]:
+        function_list: list[str, Functions] = []
+        for f in functions:
+            function_list.append(Functions(
+                identifier=f.identifier,
+                name=f.name,
+                function_type=f.function_type,
+                project_uid=f.project_uid,
+                group_addresses=f.group_addresses,
+            ))
+        return function_list
```

Comment:
consider using a list comprehension
```suggestion
        return [Functions(
                identifier=f.identifier,
                name=f.name,
                function_type=f.function_type,
                project_uid=f.project_uid,
                group_addresses=f.group_addresses,
            ) for f in functions]
```